content/blog/2022/trans-dimension-accessibility-guide/index.md

```diff
@@ -7,6 +7,7 @@ author: kim
 social: transdimension.png
 projects:
   - trans-dimension
+  - guide-to-inclusive-events
```

Comment:
Very much a content decision but just wondering if you want to link up any other blog posts related to trans dimension to the `guide-to-inclusive-events`? It just means once you've looked at the publication you'll be able to see a load of other trans dimension related content underneath it

Reply:
forgot to add other one, so will do